File: baekjoon/10818.py
n=int(input())
num=list(map(int,input().split()))
min=1000000
max=-1000000
for i in range(n):
    if num[i]>max:
        max=num[i]
    if num[i]<min:
        min=num[i]
print(min, max)

File: baekjoon/10818.c
#include <stdio.h>
int main(){
    int n,min=100,max=0;
    scanf("%d",&n);
    int num[n];
    for(int i=0;i<n;i++){
        scanf("%d",&num[i]);
        if(num[i]>max) max=num[i];
        if(num[i]<min) min=num[i];
    }
    printf("%d %d",min,max);
}